core/rawdb/freezer_table.go

```diff
@@ -823,6 +823,10 @@ func (t *freezerTable) retrieveItems(start, count, maxBytes uint64) ([]byte, []i
 			break
 		}
 	}
+
+	// Update metrics.
+	t.sizeGauge.Inc(int64(totalSize))
```

Comment:
sizeGauge is set in `newTable`, and updated in the batch commit. Updating it here should not be needed.. ? 

:+1: for updating the readMeter

Reply:
Does `sizeGauge` stand for the freeze size? If so, no need to update.

In my previous knowledge, I think this is for all read + write size

Reply:
the comment sounds this is the size of DB size 

https://github.com/ethereum/go-ethereum/blob/20f8eb756bdbfa6dbd94374bbad15883e23ab150/core/rawdb/freezer_table.go#L117